Example/Workmanager/Info.plist

```diff
@@ -22,18 +27,20 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>fetch</string>
```

Comment:
The `fetch` background mode isn't currently used in the example app, right? Is so: maybe don't enable it (yet)